catalyst/utils/metrics/mrr.py

```diff
@@ -0,0 +1,36 @@
+import torch
+
+
+def mrr(outputs: torch.Tensor, targets: torch.Tensor):
```

Comment:
could you please add docs here
https://github.com/catalyst-team/catalyst/blob/master/docs/api/utils.rst
?

Reply:
btw, what do you think about MRRCallback? like https://github.com/catalyst-team/catalyst/blob/master/catalyst/dl/callbacks/metrics/iou.py#L12 for example

Reply:
Thanks, I'll have a look at protocols.